content/notes/concepts.md

```diff
@@ -132,5 +110,5 @@ Now of course many projects or entities will seek both to some degree, however t
 
 
 * **Access**: optimizing for social welfare will mean selecting a business model which prioritizes high levels of access. By contrast, business models which use closed rights frameworks may enable you to make more money (although this is not always the case, as businesses such as Facebook demonstrate). 
```

Comment:
"you" here should be clarified e.g.

... may enable the *database provider* to make more money